app/src/main/java/io/github/androidacademyglobal/profile/ProfileScreen.kt

```diff
@@ -50,37 +55,48 @@ fun ProfileScreen(userData: ProfileScreenState) {
             .nestedScroll(scrollBehavior.nestedScrollConnection)
     ) {
         BoxWithConstraints(modifier = Modifier.weight(AppConstants.PROFILE_BOX_WEIGHT)) {
-            Surface {
-                Column(
-                    modifier = Modifier
-                        .fillMaxSize()
-                        .verticalScroll(scrollState),
-                ) {
-                    ProfileHeader(
-                        scrollState,
-                        userData,
-                        this@BoxWithConstraints.maxHeight
-                    )
-                    UserInfoFields(userData, this@BoxWithConstraints.maxHeight)
-                }
+            ProfileView(userData = userData) {
+                maxHeight
             }
         }
     }
 }
 
+@Composable
+internal fun ProfileView(
```

Comment:
It's up to you but I'd name it `ProfileScreenContent` or smth like this so not to confuse with Android `View`